### PR TITLE
Close tab switcher on clicking again

### LINKED ISSUE #1049: Clicking on tab switcher icon again should close the tab switcher
**Is your feature request related to a problem? Please describe.**
Clicking on the tab switcher icon does not do anything when the tab switcher screen is already open.

**Describe the solution you'd like**
Clicking it for the second time should close the tab switcher screen and take back to the previously open tab.

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1049
 
Changes: 

Now, the tab switcher would close and user would be taken to the previously open tab when the tab switcher button is clicked again.

GIF for the change: 

ezgif com-video-to-gif 56

### CODE REVIEW

**Comment by ShridharGoel** [comment]
@abdulwd @soloturn Please review.

**Comment by ShridharGoel** [comment]
@abdulwd Please review this

File: app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java [hunk 1]
```diff
@@ -214,6 +214,7 @@ public void onFinish() {
   private boolean wasHideToolbar = true;
   private boolean isHideToolbar = true;
   private boolean isSpeaking = false;
+  private int currentTabPosition;
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java]: I think you can use `currentWebViewIndex` instead of defining a new variable.

Comment (ShridharGoel) [reply app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java]: Done

**Comment by ShridharGoel** [comment]
Squashed.

**Pull request merged**